common/errors.js

```diff
@@ -26,6 +26,14 @@ class BaseError extends Error {
 }
 exports.BaseError = BaseError;
 
+class SshConnectionFailed extends BaseError {
+  constructor(message) {
+    super(message);
+    this.code = 'ETIMEDOUT';
```

Comment:
Put it on constant may be!

Reply:
This is already replicated many times in the same file :) 
Will put it into constant here

Reply:
Sure!